src/welle-gui/i18n/es_ES.ts

```diff
@@ -0,0 +1,1766 @@
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="184"/>
+        <source>News</source>
+        <translation>Noticias</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="185"/>
+        <source>Current Affairs</source>
+        <translation>Actualidad</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="186"/>
+        <source>Information</source>
+        <translation>Información</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="187"/>
+        <source>Sport</source>
+        <translation>Deportes</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="188"/>
+        <source>Education</source>
+        <translation>Educación</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="189"/>
+        <source>Drama</source>
+        <translation>Drama</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="190"/>
+        <source>Arts</source>
+        <translation>Cultura</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="191"/>
+        <source>Science</source>
+        <translation>Ciencia</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="192"/>
+        <source>Talk</source>
+        <translation>Varios</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="193"/>
+        <source>Pop Music</source>
+        <translation>Música pop</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="194"/>
+        <source>Rock Music</source>
+        <translation>Música rock</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="195"/>
+        <source>Easy Listening</source>
+        <translation>Música ligera</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="196"/>
+        <source>Light classical</source>
+        <translation>Clásico liviano</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="197"/>
+        <source>Classical Music</source>
+        <translation>Clásico serio</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="198"/>
+        <source>Other Music</source>
+        <translation>Otra música</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="199"/>
+        <source>Weather</source>
+        <translation>Tiempo</translation>
```

Comment:
Please use "Información Metereológica"

Reply:
Made an orthographic correction; Reason: "Información Metereológica" must be a typo in the registered tables, because the word "metereológica" does not exist, the correcto form is "meteorológica" as you can see https://dle.rae.es/meteorol%C3%B3gico

Reply:
Ok for me 😁